R/removeat.R

```diff
@@ -80,23 +80,19 @@ RemoveAt.matrix <- function(x, at = NULL, MARGIN = NULL, ignore.case = TRUE, spl
 #' @export
 RemoveAt.ftable <- function(x, at = NULL, MARGIN = NULL, ignore.case = TRUE, split = NULL)
```

Comment:
Add `@noRd`, `@rdname`, or `@describeIn` so a separate Rd file isn't created.

Reply:
This applies to all classes for this method as they are also missing it, so I'll do that to all.